modules/sriov-operator-hosted-control-planes.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_content-type: PROCEDURE
+[id="sriov-operator-hosted-control-planes_{context}"]
+= Deploying the SR-IOV Operator for hosted control planes
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
+
+[role="_abstract"]
+After you configure and deploy your hosting service cluster, you can create a subscription to the SR-IOV Operator on a hosted cluster. The SR-IOV pod runs on workers rather than control planes.
```

Comment:
Could go with either "the control plane" or "the control plane machines" here, up to you.

```suggestion
After you configure and deploy your hosting service cluster, you can create a subscription to the SR-IOV Operator on a hosted cluster. The SR-IOV pod runs on worker machines rather than the control plane.
```